app/(afterLogin)/(tna)/tna/management/[id]/_components/courseLesson/index.tsx

```
@@ -1,10 +1,12 @@
-import { Button } from 'antd';
+import { Button, Collapse, Spin } from 'antd';
```

ESLint

[error] 'Spin' is defined but never used.

(@typescript-eslint/no-unused-vars)